store/sql_channel_store.go

```diff
@@ -524,6 +524,40 @@ func (s SqlChannelStore) GetMoreChannels(teamId string, userId string, offset in
 	return storeChannel
 }
 
+func (s SqlChannelStore) GetPublicChannels(teamId string, offset int, limit int) StoreChannel {
+	storeChannel := make(StoreChannel, 1)
+
+	go func() {
+		result := StoreResult{}
+
+		data := &model.ChannelList{}
+		_, err := s.GetReplica().Select(data,
+			`SELECT
+			    *
+			FROM
+			    Channels
+			WHERE
+			    TeamId = :TeamId
+					AND Type IN ('O')
```

Comment:
Can we do `Type = 'O'` instead? I know the database engine will probably optimize it anyway but it's nice to be more specific